packages/web-ui/src/Button/Button.docs.mdx

```diff
@@ -138,18 +152,15 @@ can be used to display different sizes at different breakpoints.
 You can nest icons directly inside the `Button`. An appropriate gap is provided
```

Comment:
Could we add examples with icons on the left as well? And let's omit the word 'page' in the examples, as it suggests a specific use case. We can use the words 'next' and 'back' instead

<img width="697" alt="Screenshot 2024-09-26 at 08 59 43" src="https://github.com/user-attachments/assets/28355c47-96a7-4dc4-9660-d837cd09a289">